test/core/test_comparison.py

```diff
@@ -109,3 +109,9 @@ def test_empty_with_no_builds(self):
         new_project = self.group.projects.create(slug='new')
         comparison = TestComparison.compare_projects(new_project)
         self.assertFalse(comparison.diff)
+
+    def test_regressions(self):
+        comparison = TestComparison.compare_builds(self.build1, self.build2)
+        regressions = comparison.regressions
+        self.assertEqual(['a'], regressions['myenv'])
```

Comment:
TestComparison is used both when comparing projects _and_ when comparing builds from different projects, but this regression detection code is only by email notifications.

This test uses builds from different projects because the test data is already created in `setUp()`. I'm not keen on adding even more test setup here, but I will add comments for this effect.